server/schema/notifications/createNotification.js

```diff
@@ -0,0 +1,34 @@
+const schema = {
+  required: ['heading', 'description', 'public', 'target_usernames'],
```

Comment:
`description` should not be required as it is null by default, so it will remain null in DB unless provided some value